File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/firsttoolwin/vb/pkgcmdid.vb
' PkgCmdID.vb
Imports System

Class PkgCmdIDList
    Private Sub New()
    End Sub

        Public Const cmdidWindowsMedia As UInteger = &H100
    Public Const cmdidWindowsMediaWin As UInteger = &H101

    '<snippet11>
    Public Const cmdidWindowsMediaOpen As Integer = &H132
    Public Const cmdidWindowsMediaFilename As Integer = &H133
    Public Const cmdidWindowsMediaFilenameGetList As Integer = &H134
    Public Const ToolbarID As Integer = &H1000
    '</snippet11>

    Public Const cmdidMyCommand As Integer = &H139

End Class

File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/firsttoolwin/cs/pkgcmdid.cs
// PkgCmdID.cs
// MUST match PkgCmdID.h
using System;

namespace Company.FirstToolWin
{
    static class PkgCmdIDList
    {
        public const uint cmdidWindowsMedia = 0x100;
        public const uint cmdidWindowsMediaWin = 0x101;
//<snippet11>
        public const int cmdidWindowsMediaOpen = 0x132;
        public const int cmdidWindowsMediaFilename = 0x133;
        public const int cmdidWindowsMediaFilenameGetList = 0x134;
        public const int ToolbarID = 0x1000;
//</snippet11>
    };
}